apps/dashboard/src/components/workflow-editor/steps/push/push-editor.tsx

```diff
@@ -0,0 +1,23 @@
+import { getComponentByType } from '@/components/workflow-editor/steps/component-utils';
+import { PushTabsSection } from '@/components/workflow-editor/steps/push/push-tabs-section';
+import { type UiSchema } from '@novu/shared';
+
+const subjectKey = 'subject';
+const bodyKey = 'body';
+
+type PushEditorProps = { uiSchema: UiSchema };
+export const PushEditor = (props: PushEditorProps) => {
+  const { uiSchema } = props;
+  const { [bodyKey]: body, [subjectKey]: subject } = uiSchema?.properties ?? {};
+
+  return (
+    <div className="flex h-full flex-col">
+      <PushTabsSection className="py-5">
```

Comment:
The in-app section contains an `In-app template editor` header. Let's add it to push as well. cc: @twentyone24 

<img width="607" alt="Screenshot 2024-12-16 at 11 00 46" src="https://github.com/user-attachments/assets/7d15e98f-387b-47a0-a871-c5447394f19e" />